### PR TITLE
Refer to userinfo section in Common Topics

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
@thormarius - Here is a proposal. You can probably remove the example and refer to the common topics page about user info.

File: vipps-login-api.md [hunk 1]
```diff
@@ -185,6 +185,10 @@ Which of the flows to use is controlled with the initiation of the individual lo
 
 The flows are described below.
 
+See
```
Comment (thormarius) [inline vipps-login-api.md:188]: Not sure if I understood why you're referring to the landingpage (ecom) here?

File: vipps-login-api.md [hunk 2]
```diff
@@ -710,65 +718,9 @@ The access token is received on a successful request to the [token endpoint](#oa
...
-    "accounts": [
-        {
-            "account_name": "My savings",
-            "account_number": "12064590675",
-            "bank_name": "My bank"
-        }
-    ]
-}
-```
+See
```
Comment (thormarius) [inline vipps-login-api.md:721]: Is this almost a duplicate of the section above?

File: vipps-login-api.md (hunk 1 above)
Comment (rebekaburnett) [reply vipps-login-api.md:188]: Maybe it was an error. I don't know why I put that there. I will remove it.

**Comment by rebekaburnett** [comment]
Hi @thormarius ! Here is something I was going to attempt to reduce duplication. But it got a bit messy. Maybe you have some thoughts on the topic? Here is the "common topic": https://github.com/vippsas/vipps-developers/blob/master/common-topics/userinfo.md   Cloveras just send you a [change to that section](https://github.com/vippsas/vipps-developers/pull/231)

**Pull request closed without merging**